libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -152,6 +152,25 @@ bool IRGeneratorForStatements::visit(TupleExpression const& _tuple)
 	return false;
 }
 
+bool IRGeneratorForStatements::visit(IfStatement const& _ifStatement)
+{
+	_ifStatement.condition().accept(*this);
+	string condition = expressionAsType(_ifStatement.condition(), *TypeProvider::boolean());
+
+	if (_ifStatement.falseStatement())
+	{
+		m_code << "switch " << condition << "\n" "case 0 {\n";
+		_ifStatement.falseStatement()->accept(*this);
+		m_code << "}\n" "default {\n";
```

Comment:
Do we have preferences on how to do this?
```
  m_code << "}\n";
  m_code << "default {\n";
```
and
```
  m_code << "}\n" << "default {\n";
```
are theoretically slower and
```
  m_code << "}\ndefault {\n";
```
looks just weird, so ``"}\n" "default {\n"`` (which is equivalent to the last one) seems best to me, but I'm happy to change.

Reply:
Readable beats speed and I would also not make any assumptions about speed for string literals...